core-aam/index.html

```diff
@@ -250,6 +250,7 @@ <h3>Accessibility <abbr title="Application Programming Interfaces">APIs</abbr></
           <li><cite>User Interface Automation</cite> [[UI-AUTOMATION]]</li>
           <li><cite>ATK - Accessibility Toolkit</cite> [[ATK]] and <cite>Assistive Technology Service Provider Interface</cite> [[AT-SPI]], referred to hereafter as "ATK/AT-SPI"</li>
           <li><cite>macOS Accessibility Protocol</cite> [[AXAPI]]</li>
+          <li><cite>Android</cite> [[AccessibilityNodeInfo]]</li>
```

Comment:
This link doesn't work, it should bring us to an entry in the references section: https://w3c.github.io/core-aam/#references

However, I looked around a bit and I don't see how to do this. Also I noticed that there is a strange split between normative and informative references... which I suppose I'll make an issue about: